tests/jvm/src/test/scala/scala/meta/tests/semanticdb/ExpectSuite.scala

```diff
@@ -133,6 +134,11 @@ trait ExpectHelpers extends FunSuiteLike {
         .Settings()
         .withCacheDir(AbsolutePath(target))
         .withClasspath(Classpath(AbsolutePath(in)))
+        .withDependencyClasspath(
+          Classpath(
+            Library.scalaLibrary.classpath().entries ++
+              Library.jdk.classpath().entries
+          ))
```

Comment:
Can this be written as `Library.scalaLibrary.classpath() ++ Library.jdk.classpath()`?

Reply:
Yes it can! So much nicer.